apps/web/src/components/tiptap-ui/blockquote-button/blockquote-button.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from "react"
```

Comment:
Consider importing only the entities that you use in this component instead of the whole React library. For example:

```ts
import {useState} from "react"
```

Importing  the entire library as a namespace can lead to a larger bundle size because it includes the entire React library, even if you only use a small part of it. This could be detrimental to the developers who download these components, if they care about bundle size.